lib/components/map/itinerary-summary-overlay.tsx

```diff
@@ -152,14 +152,18 @@ const ItinerarySummaryOverlay = ({
   )
 
   if (!itins || !visible) return <></>
-  const mergedItins: ItinWithGeometry[] = addTrueIndex(
-    doMergeItineraries(itins).mergedItineraries.map(addItinLineString)
+  const indexedItins: ItinWithGeometry[] = addTrueIndex(
+    itins.map(addItinLineString)
+  )
+  const mergedItins = doMergeItineraries(indexedItins).mergedItineraries
+
+  const midPoints = mergedItins.reduce(
```

Comment:
this has an any type. I think `reduce` needs to have a type and `mergedItins` needs to be typed to not get errors.
Either type `narrative-itineraries.js` and `doMergeItineraries()` or assign a type to the variable here.